vmdb/app/models/miq_provision_configured_system_request.rb

```diff
@@ -2,6 +2,7 @@ class MiqProvisionConfiguredSystemRequest < MiqRequest
   TASK_DESCRIPTION  = 'Configured System Provisioning'
   SOURCE_CLASS_NAME = 'ConfiguredSystem'
   REQUEST_TYPES     = %w(provision_via_foreman)
+  ACTIVE_STATES     = self.base_class::ACTIVE_STATES
```

Comment:
Good question, something was failing somewhere and I added it to get around the issue as it was not my main focus at the time.  Let me take it out and determine the root cause.